deploy/kubernetes-1.13/pmem-csi-lvm.yaml

```diff
@@ -98,7 +98,19 @@ spec:
   selector:
     app: pmem-csi-controller
   ports:
-  - port: 10000
+  - port: 10000 # port inside the pod where the PMEM-CSI registry listens
+---
+# TODO: only deploy this for testing
```

Comment:
This is the part where I'd like to get feedback. In order to test the normal driver deployment, we have to expose it's csi.sock to the outside world. This is something that must not be done in production.

We need to investigate how we can minimize the number of different deployment files (lvm/direct, with/without socat, perhaps support different log levels).

In the meantime my proposal is to just add these socat parts here and also add them to pmem-csi-direct.yaml (currently missing there).

Reply:
I think I have found a clean solution after rebasing this branch onto PR #234: now the `-testing` variants are deployed in our test cluster and provides the necessary port exposure, whereas the normal deployments are completely unchanged.

Therefore I now consider this PR ready for merging.